scripts/pipeline/production_reg_citest_conf.pl

```diff
@@ -27,7 +27,17 @@
 # ---------------------- TEST CORE DATABASES -----------------------------------
 
 # FORMAT: species/alias name => [ host, db_name ]
-my $citest_core_dbs = {}; # TAG: <core_dbs_hash>
+my $citest_core_dbs = {
```

Comment:
Just checking: did you want to commit this content ?

Reply:
No, I did not... Sorry about that. I'll revert it now.